### PR TITLE
Refactoring for PHP 7.1

### PR DESCRIPTION
**Pull request by janbarasek**
Optimalize code for new standard and fix code typo.

### CODE REVIEW

File: Solaris/MoonPhase.php [hunk 1]
```diff
@@ -4,335 +4,485 @@
...
 namespace Solaris;
 
-class MoonPhase {
+class MoonPhase
+{
+
+	/**
+	 * @var int|null
+	 */
 	private $timestamp;
```
Comment (solarissmoke) [inline Solaris/MoonPhase.php]: Please change this and the other private variables to `protected` - see https://github.com/solarissmoke/php-moon-phase/issues/17

File: Solaris/MoonPhase.php [hunk 2]
```diff
@@ -4,335 +4,485 @@
...
-		$epsilon = 0.000001;	// 1E-6
-		$e = $m = deg2rad($m);
-		do {
-			$delta = $e - $ecc * sin($e) - $m;
-			$e -= $delta / ( 1 - $ecc * cos($e) );
-		}
-		while ( abs($delta) > $epsilon );
-		return $e;
+	/**
+	 * KEPLER: Solve the equation of Kepler.
```
Comment (solarissmoke) [inline Solaris/MoonPhase.php]: This docstring doesn't belong to this function.

File: Solaris/MoonPhase.php [hunk 3]
```diff
@@ -4,335 +4,485 @@
...
-		$t3 = $t2 * $t;
-
-		$nt1 = 2415020.75933 + $this->synmonth * $k
-				+ 0.0001178 * $t2
-				- 0.000000155 * $t3
-				+ 0.00033 * sin( deg2rad( 166.56 + 132.87 * $t - 0.009173 * $t2 ) );
-
-		return $nt1;
+	/**
+	 * Calculates  time  of  the mean new Moon for a given
```
Comment (solarissmoke) [inline Solaris/MoonPhase.php]: This dosctring doesn't belong to the function below it.

File: Solaris/MoonPhase.php [hunk 4]
```diff
@@ -4,335 +4,485 @@
...
-		else	// Last quarter correction
-			$pt += -0.0028 + 0.0004 * cos( deg2rad( $m ) ) - 0.0003 * cos( deg2rad( $mprime ) );
-			$apcor = true;
-		}
-		if (!$apcor)	// function was called with an invalid phase selector
-			return false;
-
-		return $pt;
+	/**
+	 * Given a K value used to determine the mean phase of
```
Comment (solarissmoke) [inline Solaris/MoonPhase.php]: Mismatched docstring

File: Solaris/MoonPhase.php [hunk 5]
```diff
@@ -4,335 +4,485 @@
...
-			$this->truephase( $k2, 0.25 ),
-			$this->truephase( $k2, 0.5 ),
-			$this->truephase( $k2, 0.75 )
-		);
-
-		$this->quarters = array();
-		foreach( $data as $v )
-			$this->quarters[] = ( $v - 2440587.5 ) * 86400;	// convert to UNIX time
+	/**
+	 * Find time of phases of the moon which surround the current date.
```
Comment (solarissmoke) [inline Solaris/MoonPhase.php]: Mismatched docstring

File: composer.json [hunk 6]
```diff
@@ -1,18 +1,35 @@
...
-        { "name": "Samir Shah", "email": "samir@rayofsolaris.net" },
-        { "name": "Erik Scheepers", "email": "e.scheepers@linkorb.com", "role": "Packaging for composer" }
-    ],
-    "require": {
-        "php": ">=5.3.0"
-    },
-    "autoload": {
-        "psr-0": { "Solaris": "" }
-    }
+	"name": "janbarasek/php-moon-phase",
```
Comment (solarissmoke) [inline composer.json]: Don't think you meant to change the name and homepage of the package?

Comment (janbarasek) [reply composer.json]: Ou, sorry! It changed it because I want make new package for internal use. In new pull-request commit it will be reverted.

**Comment by solarissmoke** [comment]
Closing in favour of  #20

**Pull request closed without merging**